recipe/meta.yaml

```diff
@@ -37,11 +37,10 @@ requirements:
     - libcurl 7.88.1
     - draco 1.5.6
     - geotiff 1.7.0
-    - hdf5 1.10.6              # [not (osx and arm64)]
-    - hdf5 1.12.1              # [osx and arm64]
+    - hdf5 1.12.1
     - libgdal 3.6.2
     - libkml 1.3.0
-    - libpq 12.9
+    - libpq 12.15
```

Comment:
Is there a reason for updating the min version to 12.15?

Reply:
Because we updated postgres while building for OpenSSL 3.